docker/test/integration/minifi/core/NifiContainer.py

```diff
@@ -23,12 +23,13 @@
 
 
 class NifiContainer(FlowContainer):
-    NIFI_VERSION = '1.7.0'
+    NIFI_VERSION = '1.15.3'
```

Comment:
is this the latest version? should we offer some level of backwards compatibility?

Reply:
Yes this is the latest version currently. We have backwards compatibility as it is working with the older image as well, the changes done only serve the requirements of the newer image, so it should work with the older version as well. Or did you think about something else regarding backwards compatibility?

Reply:
I'm not sure what the lowest version of nifi we aim to support, but IMO there should be some tests that use that one

Reply:
Well the previous version we used was 4 years old, and if we work with that, then I don't think we have a lot to worry about :) I think these site to site and invokehttp use cases we test did not really change since then. If they change with a newer version later then I think we can introduce a test with an older version to verify both.

Reply:
Let's aim for the latest version and avoid breaking interop unless there is a good reason.

Reply:
I would wait for 1.16.0 to be out and update to that

Reply:
Updated to 1.16.0 in bd25b5058b3cae4a727e2cacb914bd10266e1b83

Reply:
It's not out yet. I assume someone discovered issues in the background and we're going to see another RC.

Reply:
It's strange then that the apache/nifi:1.16.0 image was already released on dockerhub 3 days ago. Anyway we can wait for the official release for the merge, if the image changes it will be pulled again automatically due to the hash change.

Reply:
fair, that's good enough for me